circuits/cpp/src/aztec3/circuits/rollup/components/components.cpp

```diff
@@ -229,7 +191,7 @@ std::array<fr, 2> compute_calldata_hash(std::array<fr, 4> calldata_hashes)
  */
 std::array<fr, 2> compute_calldata_hash(std::array<abis::PreviousRollupData<NT>, 2> previous_rollup_data)
 {
-    return compute_calldata_hash({ previous_rollup_data[0].base_or_merge_rollup_public_inputs.calldata_hash[0],
+    return accumulate_sha256<NT>({ previous_rollup_data[0].base_or_merge_rollup_public_inputs.calldata_hash[0],
```

Comment:
No the naming change was made simply because that function's functionality started to be useful even when computing the logs hash in kernel and not just when computing calldata hash in rollup circuits.